docs/modules/ROOT/pages/reference/framework-requirements.adoc

```diff
@@ -0,0 +1,46 @@
+= Framework Requirements
+
+As discussed on xref:explanations/what_is.adoc["What is it?"], a *framework* for self-service provisioning of services out of a Kubernetes cluster is a core part of the application catalog.
+
+This page discusses requirements the framework must adhere to:
+
+Reusability::
+Base parts of the framework should be reusable by the Open Source community and VSHNeers.
+When we develop a base part of the framework which might be usable by others, we try to bring it upstream or contribute it to the place it best belongs to.
+Just like Puppet base module or Helm charts are (can be) reusable.
+
+Low barrier to entry - xref:reference/glossary.adoc#_personas[Service Maintainer]::
+Adding a new service to the catalog using the framework should be as easy as possible.
+Required skills:
+Must:::
+* Kubernetes
+* Helm, Helm Charts
+* Prometheus, PromQL
+* (Crossplane)
+Should:::
+* Commodore Components
+
+Low barrier to entry - xref:reference/glossary.adoc#_personas[Service User]::
+Using a service should be a no-brainer:
+* Simple and discoverable interface
```

Comment:
What does "discoverable" mean exactly?

Reply:
For example its self-documenting so that something like "kubectl explain" works.